### PR TITLE
khal: add German translation

### PR DESCRIPTION
**Pull request by Lodifice**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/khal.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Eine textbasierte Kalender- und Planungsanwendung für die Kommandozeile.
+> Weitere Informationen: <https://lostpackets.de/khal>.
+
+- Starte khal im interaktiven Modus:
+
+`ikhal`
+
+- Gib alle Termine aus, die im Kalender "privat" in den nächsten sieben Tagen geplant sind:
+
+`khal list -a {{privat}} {{today}} {{7d}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/khal.md:12]:
```suggestion
`khal list -a privat today 7d`
```

Optional change: no such style guide requirement exists.

File: pages.de/common/khal.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Gib alle Termine aus, die in Kalendern außer "privat" für morgen um 10 Uhr geplant sind:
+
+`khal at -d {{privat}} {{tomorrow}} {{10:00}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/khal.md:16]:
```suggestion
`khal at -d privat tomorrow 10:00`
```


Optional change: no such style guide requirement exists.

File: pages.de/common/khal.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Gib einen Kalender mit einer Liste an Terminen für die nächsten drei Monate aus:
+
+`khal calendar`
+
+- Füge dem Kalender "privat" einen neuen Termin hinzu:
+
+`khal new -a {{privat}} {{2020-09-08}} {{18:00}} {{18:30}} "{{Zahnarzttermin}}"`
```
Comment (EmilySeville7cfg) [inline pages.de/common/khal.md:24]:
```suggestion
`khal new -a privat 2020-09-08 18:00 18:30 "Zahnarzttermin"`
```


Optional change: no such style guide requirement exists.

**Review by EmilySeville7cfg — APPROVED** [review]
- Approved because just optional fixes were found.
- This page should be reviewed by German native speakers.
- Thanks for the contribution. 😸

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**